docs/running_psalm/issues/MissingConstructor.md

```diff
@@ -1,6 +1,16 @@
 # MissingConstructor
 
-Emitted when non-null properties without default values are defined in a class without a `__construct` method
+Unitialized properties are statically hard to analyze. To prevent mistakes, Psalm will enforce that all properties should be initialized.
```

Comment:
"statically hard to analyze" -> "hard to statically analyze"?
"properties should be" -> "properties must be" (maybe?)